src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsCounter = 0;
+
+  const getSum = (a) => {
+    callsCounter++;
+
+    if (callsCounter > 3 && callsCounter % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    }
+
+    return b => a + b;
```

Comment:
you should return the function only once here (a function inside of a `getSum` function), and return the results inside of it only